src/menu/menu-content.styles.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import { Border, Breakpoint, Colour, Radius } from "../theme";
+import { overflow } from "../popover-v2/types";
+
+// =============================================================================
+// STYLES INTERFACE
+// =============================================================================
+interface MenuPanelStylesProps {
+    $overflow?: overflow | undefined;
+    $maxHeight?: number | undefined;
+}
+
+// =============================================================================
+// MENU CONTENT STYLES
+// =============================================================================
+export const MenuPanel = styled.div<MenuPanelStylesProps>`
```

Comment:
would be good to set the scrollbar style too. this is used for dropdown

```
&::-webkit-scrollbar {
    width: 14px;
}

&::-webkit-scrollbar-track {
    background: transparent;
}

&::-webkit-scrollbar-thumb {
    background: ${Colour["bg-inverse-subtlest"]};
    border: 5px solid transparent;
    border-radius: ${Radius["full"]};
    background-clip: padding-box;
}
```

Reply:
udpate. also updated the popoverCard for default popovercontent